lib/loggers/index.js

```diff
@@ -185,10 +147,22 @@ container.init = async () => {
   if(cluster.isMaster) {
     // create shared transports
     const transports = container.transports;
-    transports.console = new ModuleConsoleTransport(config.loggers.console);
-    transports.app = new ModuleFileTransport(config.loggers.app);
-    transports.access = new ModuleFileTransport(config.loggers.access);
-    transports.error = new ModuleFileTransport(config.loggers.error);
+    transports.console = new winston.transports.Console({
+      ...config.loggers.console,
+      format: formatters.fromConfig(config.loggers.console.bedrock),
+    });
+    transports.app = new winston.transports.File({
+      ...config.loggers.app,
+      format: formatters.fromConfig(config.loggers.app.bedrock),
+    });
+    transports.access = new winston.transports.File({
+      ...config.loggers.access,
+      format: formatters.fromConfig(config.loggers.access.bedrock),
+    });
+    transports.error = new winston.transports.File({
+      ...config.loggers.error,
+      format: formatters.fromConfig(config.loggers.error.bedrock),
+    });
```

Comment:
I'm just going to presume the above changes are winston 3.x equivalents.

Reply:
The `default` formatter here gives us parity with the current console logging we know and love.